src/lib/import-export/import/handlers/backup-handler-zip.ts

```diff
@@ -9,7 +9,12 @@ export class BackupHandlerZip extends EventEmitter implements BackupHandler {
 		const zip = new AdmZip( backup.path );
 		return zip
 			.getEntries()
-			.map( ( entry ) => entry.entryName )
+			.map( ( entry ) => {
+				if ( entry.entryName.startsWith( '/' ) ) {
```

Comment:
Tbh, I have no idea. I extracted the zip file but no success founding out why is that.
I guess, maybe it's because it's a zip and not a tar, and as we used a different library for zip and tar, it has different outcomes.
But it's only a guess, and probably it's not correct.